drf/accounts/api/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib import admin
```

Comment:
You don't need api folder inside app because we are building rest service. without Django default front part.